IanGRap/Squish
# New font

Contributors: Alfredo

Goal: Update the old fonts to Knewave. Make sure that the Squish font is only used for the Squish logo.

Changes: 
- Created two Knewave font assets. One is made to mimic the Squish logo in the coloring. This is used in the Score Screen and in the Win Screen. 
- Updated Prefab UIManager
- Title Screen, Pause Screen, Win Screen
- Lobby: Didn't change the images with letters
- Score Screen: I made two versions of the Knewave font. The scores still have the green purple glow

**Note**: I didn't update the settings Font because I know that Austin is currently changing it. 

How to Test: 
- Play through the whole game from start to finish. Look at the fonts.
- Did I miss anything? Does the colored version of the Knewave font look too much like the Squish logo?

--- 
I added an outline and drop shadow to the default Knewave asset based on Ian's recommendations. This should make it more visible in the lobby against bright backgrounds.
